### PR TITLE
Load game option save config

### PR DESCRIPTION
**Pull request by xiaoyifang**
image

### CODE REVIEW

**Comment by xiaoyifang** [comment]
image
have no right to set assigner

File: katrain/gui/popups.py [hunk 1]
```diff
@@ -777,6 +777,9 @@ def __init__(self, **kwargs):
     def on_submit(self):
         self.filesel.button_clicked()
 
+    def update_config(self, save_to_file=True, close_popup=False):
```
Comment (sanderland) [inline katrain/gui/popups.py:780]: this is redundant, right?

File: katrain/popups.kv [hunk 2]
```diff
@@ -753,14 +753,17 @@
             LabelledCheckBox:
                 id: fast
                 size_hint: 0.05,1
+                input_property: "general/load_fast_analysis"
+                on_press: root.update_config(True)
```
Comment (sanderland) [inline katrain/popups.kv:757]: could we do this only on submit?

**Review by sanderland — CHANGES_REQUESTED** [review]
nice idea!

File: katrain/popups.kv [hunk 3]
```diff
@@ -753,14 +753,17 @@
...
             DescriptionLabel:
                 text: i18n._("load sgf rewind")
                 halign: 'right'
                 size_hint: 0.45,1
             LabelledCheckBox:
                 id: rewind
-                active: True
                 size_hint: 0.05,1
+                input_property: "general/load_sgf_rewind"
```
Comment (sanderland) [inline katrain/popups.kv:765]: config.json should also have these fields

File: katrain/popups.kv (hunk 2 above)
Comment (xiaoyifang) [reply katrain/popups.kv:757]: you mean on load file?

File: katrain/gui/popups.py (hunk 1 above)
Comment (xiaoyifang) [reply katrain/gui/popups.py:780]: yes

File: katrain/popups.kv (hunk 2 above)
Comment (sanderland) [reply katrain/popups.kv:757]: yep, just when the dialog is submitted. a little more consistent.

File: katrain/popups.kv (hunk 3 above)
Comment (xiaoyifang) [reply katrain/popups.kv:766]: not needed in the config.json.
the checkbox has default value.

when user change the check state,the field will be write back into the config file.

File: katrain/popups.kv (hunk 2 above)
Comment (xiaoyifang) [reply katrain/popups.kv:757]: i am not familiar with the dialog event ,
when debugging ,the on_submit method was not invoked. any idea?

Comment (xiaoyifang) [reply katrain/popups.kv]: done

File: katrain/popups.kv (hunk 3 above)
Comment (xiaoyifang) [reply katrain/popups.kv:765]: anyway ,added in the config file.

**Comment by sanderland** [comment]
slight adjustment in 28ffa1d

**Pull request merged**